package.json

```diff
@@ -22,8 +22,14 @@
   "devDependencies": {
```

Comment:
Use fixed version for all dependencies. You can actually configure npm to use fixed version by default: in ~/.npmrc add `save-exact=true`